third_party/terraform/website/docs/r/container_node_pool.html.markdown

```diff
@@ -8,13 +8,46 @@ description: |-
 
 # google\_container\_node\_pool
 
-Manages a Node Pool resource within GKE. For more information see
-[the official documentation](https://cloud.google.com/container-engine/docs/node-pools)
-and
-[API](https://cloud.google.com/container-engine/reference/rest/v1/projects.zones.clusters.nodePools).
+Manages a node pool in a Google Kubernetes Engine (GKE) cluster. For more
+information see [the official documentation](https://cloud.google.com/container-engine/docs/node-pools)
+and [the API reference](https://cloud.google.com/container-engine/reference/rest/v1/projects.zones.clusters.nodePools).
+
+### Example Usage - Without a default pool (recommended)
+
+```hcl
+resource "google_container_cluster" "primary" {
+  name   = "my-gke-cluster"
+  region = "us-central1"
+  
+  # We can't create a cluster with no node pool defined, but we want to only use
+  # explicitly defined node pools. So, we create the smallest possible default
+  # node pool and immediately delete it.
+  remove_default_node_pool = true
+  initial_node_count = 1
+}
+
+resource "google_container_node_pool" "primary_preemptible_nodes" {
+  name       = "my-node-pool"
+  region     = "us-central1"
+  cluster    = "${google_container_cluster.primary.name}"
+  node_count = 1
+
+  node_config {
+    preemptible  = true
+    machine_type = "n1-standard-1"
+
+    oauth_scopes = [
+      "compute-rw",
+      "storage-ro",
+      "logging-write",
+      "monitoring",
+    ]
+  }
+}
+```
+
+### Example Usage - mixed usage with a default pool and split out node pool
```

Comment:
Woah, reading quickly through the examples didn't make it obvious at all that this creates 2 different node pools! Maybe you can be extra explicit here that there are 2 pools because if I didn't notice the title changing here I'd blow right past this. Maybe "mixed usage: single cluster with 2 pools: default pool and separately managed node pool" or just clarify below in comments.